helm/values.yaml

```diff
@@ -15,6 +15,9 @@ image:
 imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
+healthProbePort: 8081
+metricsPort: 8080
+webhookPort: 9443
```

Comment:
can you please write a small comment about what we are confugiring here? 
For example,
```yaml
# healthProbePort identifies port for ....
healthProbePort: 8081
````

Reply:
I've added comments for the port variables